CMakeLists.txt

```diff
@@ -0,0 +1,228 @@
+
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -Wno-unused-parameter -fdata-sections -ffunction-sections -fno-show-column -std=c++11")
+
+  # Wrap libraries in --start-group and --end-group to easily support static linking and symbol resolution, maybe useful on APPLE also, but I don't know
+  string(REPLACE "<LINK_LIBRARIES>" "-Wl,--gc-sections -Wl,--start-group <LINK_LIBRARIES> -Wl,--end-group" CMAKE_C_LINK_EXECUTABLE "${CMAKE_C_LINK_EXECUTABLE}")
+  string(REPLACE "<LINK_LIBRARIES>" "-Wl,--gc-sections -Wl,--start-group <LINK_LIBRARIES> -Wl,--end-group" CMAKE_CXX_LINK_EXECUTABLE "${CMAKE_CXX_LINK_EXECUTABLE}")
+  string(REPLACE "<LINK_LIBRARIES>" "-Wl,--start-group <LINK_LIBRARIES> -Wl,--end-group" CMAKE_C_CREATE_SHARED_LIBRARY "${CMAKE_C_CREATE_SHARED_LIBRARY}")
+  string(REPLACE "<LINK_LIBRARIES>" "-Wl,--start-group <LINK_LIBRARIES> -Wl,--end-group" CMAKE_CXX_CREATE_SHARED_LIBRARY "${CMAKE_CXX_CREATE_SHARED_LIBRARY}")
+  string(REPLACE "<LINK_LIBRARIES>" "-Wl,--start-group <LINK_LIBRARIES> -Wl,--end-group" CMAKE_C_CREATE_SHARED_MODULE "${CMAKE_C_CREATE_SHARED_MODULE}")
+  string(REPLACE "<LINK_LIBRARIES>" "-Wl,--start-group <LINK_LIBRARIES> -Wl,--end-group" CMAKE_CXX_CREATE_SHARED_MODULE "${CMAKE_CXX_CREATE_SHARED_MODULE}")
+elseif(CMAKE_HOST_WIN32)
+
+else()
+  message(STATUS "Unknown platform, maybe not supported")
+endif()
+
+# platform specific warning settings
+if(CMAKE_HOST_WIN32)
+  if(CMAKE_CXX_FLAGS MATCHES "/W[0-4]")
+    string(REGEX REPLACE "/W[0-4]" "/W4" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+  else()
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4")
+  endif()
+else()
+  set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -Wall -Wno-multichar -Wpointer-arith -Wshadow")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-multichar -Wpointer-arith -Wshadow -Wnon-virtual-dtor")
+endif()
+
+
+# header detection, could get rid of it, but it is used by lua as well
+include(CheckIncludeFile)
+function(check_has_header FILE_NAME PP_NAME)
+  check_include_file(${FILE_NAME} ${PP_NAME})
+  if(${PP_NAME})
+    add_definitions(-D${PP_NAME}=1)
+  endif()
+endfunction()
+
+check_has_header("stdint.h"    HAVE_STDINT_H)
+check_has_header("inttypes.h"  HAVE_INTTYPES_H)
+check_has_header("sys/types.h" HAVE_SYS_TYPES_H)
+
+
+# endianess detection, could be replaced by including Boost.Config 
+include(TestBigEndian)
+test_big_endian(XOREOS_BIG_ENDIAN)
+if(XOREOS_BIG_ENDIAN)
+  add_definitions(-DXOREOS_BIG_ENDIAN=1)
+else()
+  add_definitions(-DXOREOS_LITTLE_ENDIAN=1)
+endif()
+
+
+# -------------------------------------------------------------------------
+# subfolders where built binaries and libraries will be located, relative to the build folder
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+
+
+# -------------------------------------------------------------------------
+# find the required libraries
```

Comment:
You're missing liblzma, which is used for BZF archives (which are found in the iOS version of KotOR)